src/molecules/formfields/CheckboxList.js

```diff
@@ -0,0 +1,46 @@
+import React, { PropTypes } from 'react';
+import styles from './formfields.module.scss';
+import classnames from 'classnames';
+
+function CheckboxList ( props ) {
+  const {
+    label,
+    focused,
+    data,
+    handleClick
+  } = props;
+
+  return (
+    <div className={ classnames(styles.checkboxList, { [styles.focused]: focused })}>
+      <label className={ styles.label }>{ label }</label>
+      <div className={ styles.content }>
+        {data.map((item, idx) => {
+          return (
+            <label className={ styles.checkbox } key={ idx } onClick={ () => handleClick( item.id ) }>
```

Comment:
just curious why you inlined the `onClick` function? I feel like we should just pass a callback via props to give the end user more flexibility? at least for something like a checkbox, where the callback can register the event and the end user can manipulate that as they wish